NOAA-FIMS/FIMS
# #90 cleanup readme add badges and fix inith issue

Thanks for opening a pull request (PR) to FIMS! Please describe the changes under the headings below. Please also go through the checklist below to ensure the appropriate steps have been taken during development. For smaller changes, feel free to skip steps that aren't relevant, but at least describe the feature under the "what is the feature".

# What is the feature?
This pull request addresses issue #90. The commits clean up the repo and added codecov for R and C++ tests. However, we have not updated the doxygen documentation. Can we create a new issue for that task? 

# How have you implemented the solution?
Following changes have been made to the repo.
- Add URLs to the DESCRIPTION file
- Set up codecov for R tests
- Set up codecov for C++ tests
-  Add a coverage and r-cmd-check badge (also lifecycle?) to README
-  Add installation instructions to README
-  Change init.hpp to init.h
-  remove a.exe
-  add .vscode to .gitignore
-  remove -w -O3 flags

# Does it impact any other area of the project?
No.

# Does this change impact the model input or output?*
No

# How to test this change
All [GHA checks](https://github.com/NOAA-FIMS/FIMS/actions?query=branch%3A90-cleanup-readme-add-badges-and-fit-inith-issue) have passed. 
# Developer pre-PR Checklist

Please do these steps locally for big changes. Note GitHub Actions does all of these checks automatically when pushing to the repository. Please see [code development section in the contributor guide](https://noaa-fims.github.io/collaborative_workflow/contributor-guidelines.html#code-development) for details.
